.github/workflows/ci.yaml

```diff
@@ -33,7 +33,7 @@ jobs:
         run: |
           cd client
           npm install
-          npm run check-format
+          eslint . --ext .js,.jsx,.ts,.tsx
```

Comment:
Right now this is failing 
![image](https://user-images.githubusercontent.com/70448914/206010029-66f6fff1-ce41-454b-b4c5-02e4062e4aa8.png)